CWE-94-Python_darwin/DennisMitchell_m/m.py

```
@@ -0,0 +1,1631 @@
+	return [link.arity for link in links]
+
+def create_chain(chain, arity):
+	return attrdict(
+		arity = arity,
+		call = lambda x = None, y = None: variadic_chain(chain, (x, y))
+	)
+
+def create_literal(string):
+	return attrdict(
+		arity = 0,
+		call = lambda: listify(eval(string))
```

Ruff (0.13.2)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)